lib/services/sku-pack-service.js

```diff
@@ -371,6 +371,22 @@ function skuPackServiceFactory(
             });
     };
 
+    SkuPackService.prototype.getPackInfo = function(skuid) {
+        var self = this;
+        return self.loader.getAll(self.confRoot)
+        .then(function (conf) {
+            var skuConf = conf[skuid + '.json'];
+            if(skuConf) {
+                var json = JSON.parse(skuConf);
+                return {
+                    description: json.description || null,
+                    version: json.version || null
+                };
+            }
+            return null;
```

Comment:
Since in line  382-383 you defaults the `description` and `version` to null, so why not return {description: null, version: null} here? I think this will match with your previous code.